packages/api/app/Models/Role.js

```diff
@@ -0,0 +1,14 @@
+/** @type {typeof import('@adonisjs/lucid/src/Lucid/Model')} */
+const Model = use('Model');
+
+class Role extends Model {
+	users() {
```

Comment:
Não seria o caso de ter esse relacionamento a partir do usuário? Aqui seria `belongsToMany` e no usuário seria `hasOne`

Assim a partir de usuario eu poderia facilmente pegar a Role associad ao usuário.

Reply:
Discutimos na reunião: `hasMany` tá certo, precisa só colocar o `hasOne`

Reply:
Eu testei usando o hasOne, infelizmente não dá certo, o correto é apenas o hasMany in Roles, o hasOne só pode ser usado numa relação One to One.